tests/test_experimental/test_time_series.py

```diff
@@ -19,6 +21,16 @@ def ts_name():
 
 
 class TestTimeseries:
+    @pytest.fixture(scope="class")
+    def created_ts_id(self):
+        tso = TimeSeries(TS_NAME)
```

Comment:
This looks good, but there is really no need for the ts_name fixture. Seems it was just lying around from some old code. Just generate the random name in this fixture, and move it outside the test class. Also, then it will not need the class scope.